app/services/source-filters.ts

```diff
@@ -118,10 +118,10 @@ export class SourceFiltersService extends Service {
     const source = this.sourcesService.getSource(sourceId);
     const obsFilter = obs.FilterFactory.create(filterType, filterName);
     source.getObsInput().addFilter(obsFilter);
+    if (settings) obsFilter.update(settings);
```

Comment:
This was an issue with filters copy/paste?

Reply:
He just moved it up above the release.  The issue was that IPC didn't like updating the settings on a technically released reference, which worked fine in OSN.

Reply:
Yes it was causing the copy/paste not to work properly. The filter was getting copies just fine but the settings were not ported over.